runtime/src/vm/memory.rs

```diff
@@ -305,7 +403,7 @@ impl HandlerMemory {
         let mut frac = HandlerMemory::new(None, 1);
         frac.write_fixed(0, arr.mem.pop().unwrap());
         // This is a really shitty side-channel signal that it's fixed data
```

Comment:
is this still shitty though or ideally how would we do it?

Reply:
I'm not sure, yet, but I don't like the idea of a HandlerMemory object being spawned to house a single integer.